src/fides/api/alembic/migrations/versions/c90560a61f04_add_google_cloud_sql_mysql_to_.py

```diff
@@ -0,0 +1,46 @@
+"""add google_cloud_sql_mysql to connectiontype
+
+Revision ID: c90560a61f04
+Revises: 5fe01e730171
+Create Date: 2024-06-12 15:30:16.329849
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "c90560a61f04"
+down_revision = "5fe01e730171"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add google_cloud_sql_mysql to ConnectionType enum
+    op.execute("alter type connectiontype rename to connectiontype_old")
+    op.execute(
+        "create type connectiontype as enum('postgres', 'mongodb', 'mysql', 'https', 'snowflake', 'redshift', 'mssql', 'mariadb', 'bigquery', 'saas', 'manual', 'email', 'manual_webhook', 'timescale', 'fides', 'sovrn', 'google_cloud_sql_mysql')"
+    )
```

Comment:
There's still a few missing values, make sure to include all of the values from the `ConnectionType` enum https://github.com/ethyca/fides/blob/63e87c7320aa97d5bea153fe4bd9c38cae0cc759/src/fides/api/models/connectionconfig.py#L37-L55

So the complete statement should be
```
CREATE TYPE connectiontype AS ENUM ('attentive', 'bigquery', 'dynamodb', 'fides', 'generic_consent_email', 'generic_erasure_email', 'google_cloud_sql_mysql', 'https', 'manual', 'manual_webhook', 'mariadb', 'mongodb', 'mssql', 'mysql', 'postgres', 'redshift', 'saas', 'snowflake', 'sovrn', 'timescale');
```

And in the downgrade
```
CREATE TYPE connectiontype AS ENUM ('attentive', 'bigquery', 'dynamodb', 'fides', 'generic_consent_email', 'generic_erasure_email', 'google_cloud_sql_mysql', 'https', 'manual', 'manual_webhook', 'mariadb', 'mongodb', 'mssql', 'mysql', 'postgres', 'redshift', 'saas', 'snowflake', 'sovrn', 'timescale');
```